BootloaderCorePkg/Tools/BuildUtility.py

```diff
@@ -36,7 +36,7 @@
     'python'    : '3.6.0',
     'nasm'      : '2.12.02',
     'iasl'      : '20160422',
-    'openssl'   : '1.1.0g',
+    'openssl'   : '1.0.2k',
```

Comment:
Our project requires us to use newer openssl (at least 1.1.0).   It is part of the security requirement, and it just tries to avoid any potential vulnerability in the old openssl.  So might be hard to do this change on our side.

Reply:
OK, I understand.  Thank you!

Reply:
Thank you for understanding!